posthog/models/plugin.py

```diff
@@ -64,3 +67,21 @@ class Meta:
     plugin_config: models.ForeignKey = models.ForeignKey("PluginConfig", on_delete=models.CASCADE)
     key: models.CharField = models.CharField(max_length=200)
     value: models.TextField = models.TextField(blank=True, null=True)
+
+
+class PluginLogEntry(UUIDModel):
+    class Type(models.TextChoices):
+        INFO = "INFO", "info"
+        WARN = "WARN", "warn"
+        ERROR = "ERROR", "error"
+
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE, null=True)
+    plugin: models.ForeignKey = models.ForeignKey("Plugin", on_delete=models.CASCADE)
+    timestamp: models.DateTimeField = models.DateTimeField(default=timezone.now)
+    type: models.CharField = models.CharField(max_length=20, choices=Type.choices)
+    message: models.CharField = models.CharField(max_length=500)
```

Comment:
is the 500 char limit reasonable? It could be a textfield perhaps?

Reply:
Done.